### PR TITLE
Colors and More

### PR DESCRIPTION
**Pull request by dshanske**
This adds colors for the icons added in the previous release, adds Indieweb Press This and Simple Location to the suggested plugins, adds PGP key support for Indieauth and updates the Getting Started instructions.

### CODE REVIEW

**Comment by miklb** [comment]
As we discussed, until there's a solution for the Press-This plugin, I would omit it from the suggested plugins. Otherwise everything looks ok to me.

I would advocate for black&white version of icons, but that could be a future feature. Theme devs can un-enqueue the stylesheet and style their own or somehow otherwise override the SVG colors.

File: includes/class-hcard-user.php [hunk 1]
```diff
@@ -459,22 +461,33 @@ public static function relme_head_list( $author_id = null ) {
...
+		global $authordata;
+		$single_author = get_option( 'iw_single_author' );
+		if ( is_front_page() && 1 === (int) $single_author ) {
+			$author_id = get_option( 'iw_default_author' ); // Set the author ID to default
+		} elseif ( is_author() ) {
+			$author_id = $authordata->ID;
+		}
+		$pgp = get_user_option( 'pgp', $author_id );
+		if ( ! empty( $pgp ) ) {
+			echo '<link rel="pgpkey" href="' . $php . '">';
```
Comment (miklb) [inline includes/class-hcard-user.php]: typo on the `$pgp` variable reads `$php`

**Comment by dshanske** [comment]
I have a setting in Syndication Links that doesn't enqueue the color stylesheet, I could add it as a feature here.

File: includes/getting-started.php [hunk 2]
```diff
@@ -65,7 +42,7 @@
 			target="_blank">SemPress</a> is the only theme in the WordPress repository that is fully
 			microformats2 compliant. <a href="http://wordpress.org/themes/independent-publisher/"
 			target="_blank">Independent Publisher</a> has been updated to include basic microformats2 and
-			webmention display support. In practice, most themes will work relatively well out of the box, though there can be some minor display issues.', 'indieweb'
+			webmention display support. In practice, most themes will work relatively well out of the box, though there can be some minor display issues. If you are interested in receiving help in converting your theme to support Microformats 2, the community will be happy to assist.', 'indieweb'
```
Comment (miklb) [inline includes/getting-started.php]: Perhaps a link to the indieweb.org/discuss page for indieweb-wordpress on the "community will be happy to assist"?

File: languages/wordpress-indieweb.pot [hunk 3]
```diff
@@ -268,153 +268,88 @@ msgid ""
...
+"single author or multi-author site"
 msgstr ""
 
-#: includes/getting-started.php:35
+#: includes/getting-started.php:18
 msgid ""
-"Linking to the syndicated version of a post allows for bridgy to be able to "
-"discover the post as well as for end users to comment on those sites as a "
-"way of replying."
+"Links to your various sites will now appear invisibly on your site, "
```
Comment (miklb) [inline languages/wordpress-indieweb.pot]: maybe clarify invisibly with what it does. Something like "Links to your various sites are now inserted into the HTML <head> element."

File: languages/wordpress-indieweb.pot [hunk 4]
```diff
@@ -268,153 +268,88 @@ msgid ""
...
 msgstr ""
 
-#: includes/getting-started.php:47
+#: includes/getting-started.php:26
 msgid ""
-"<strong>Indieweb Press This</strong> - An Indiewebified version of Press "
-"This"
+"You can now install the Bridgy plugin. It will add its settings options "
+"under Indieweb and allow you to directly register for Bridgy to connect "
+"your blog to responses from sites such as Facebook and Twitter too"
```
Comment (miklb) [inline languages/wordpress-indieweb.pot]: remote too

File: languages/wordpress-indieweb.pot [hunk 5]
```diff
@@ -268,153 +268,88 @@ msgid ""
...
 msgstr ""
 
-#: includes/getting-started.php:57
+#: includes/getting-started.php:34
 msgid ""
-"Some WordPress themes are compatible with microformats. The IndieWeb uses "
-"microformats2, the latest version, to mark up sites so that they can be "
-"interpreted by other sites when retrieved. Most parsers will fall back onto "
-"the older format if available."
+"The IndieWeb uses microformats2 to mark up sites so that they can be "
```
Comment (miklb) [inline languages/wordpress-indieweb.pot]: Should this have a link to an IW wiki page for mf2? Is there a better introduction to mf2 somewhere else that doesn't get into plumbing?

File: languages/wordpress-indieweb.pot [hunk 6]
```diff
@@ -268,153 +268,88 @@ msgid ""
...
-#: includes/getting-started.php:58
+#: includes/getting-started.php:35
 msgid ""
-"If your theme is not compatible, the Microformats 2 Plugin attempts to add "
-"microformats to a site that does not support them. This is not a "
-"replacement for a correctly configured theme. If you are interested in "
-"receiving help in converting your theme to support Microformats 2, the "
-"community will be happy to assist."
+"If your theme does not identify itself as supporting Microformats 2, the "
+"Microformats 2 Plugin attempts to add microformats to a site that does not "
```
Comment (miklb) [inline languages/wordpress-indieweb.pot]: add mf to a theme that doesn't support them

File: readme.md [hunk 7]
```diff
@@ -68,10 +68,11 @@ If you think we missed a plugin reference, please file an issue on [Github](http
 * Semantic Linkbacks (Required) - makes IndieWeb comments and mentions look better on your site.
```
Comment (miklb) [inline readme.md]: possible alternative description(?)- Interprets Webmentions and converts them to comments and reactions that you can display in your theme.

File: readme.md [hunk 8]
```diff
@@ -68,10 +68,11 @@ If you think we missed a plugin reference, please file an issue on [Github](http
...
 * Post Kinds - Allows you to reply/like/RSVP etc to another site from your own, by adding support for kinds of posts to WordPress.
 * Syndication Links - Adds fields to a post to allow manual entry of syndication links as well as automatically from a supported syndication plugin. Fully supports Social, partial support for NextScripts: Social Networks Auto-Poster (aka SNAP).
-* MicroPub - A MicroPub Server
+* MicroPub - A MicroPub Server for publishing posts using Micropub clients
```
Comment (miklb) [inline readme.md]: would the inclusion of "third party" Micropub clients help people new to this understand that it requires another tool to publish?

File: languages/wordpress-indieweb.pot (hunk 3 above)
Comment (dshanske) [reply languages/wordpress-indieweb.pot]: Too technical, I think. But I'll change it slightly.

**Review by miklb — APPROVED** [review]

**Pull request merged**